ladybug/src/main/java/nl/nn/ibistesttool/IbisDebuggerAdvice.java

```diff
@@ -368,11 +368,29 @@ public ThreadDebugInfo announceChildThread(Object owner, String correlationId) {
 	}
 
 	@Override
-	public Object threadCreated(Object handle, Object request) {
+	public void cancelChildThread(ThreadDebugInfo threadInfo) {
+		if (!isEnabled()) {
+			return;
+		}
+		// create dummy thread to be able to start and end child thread, to avoid ladybug waiting for thread to start/end that will not happen.
+		// TODO replace with call to Ladybug cancelChildThread() when available.
+		Thread thread = new Thread() {
```

Comment:
Is hier een kaartje voor gemaakt aan de ladybug kant?

Reply:
https://github.com/ibissource/ibis-ladybug/issues/62